packages/dd-trace/src/debugger/devtools_client/snapshot/collector.js

```diff
@@ -10,6 +10,28 @@ module.exports = {
   getRuntimeObject: getObject
 }
 
+/**
+ * @typedef {Object} GetObjectOptions
+ * @property {Object} maxReferenceDepth - The maximum depth of the object to traverse
+ * @property {number} maxCollectionSize - The maximum size of a collection to include in the snapshot
+ * @property {number} maxFieldCount - The maximum number of properties on an object to include in the snapshot
+ * @property {bigint} deadlineNs - The deadline in nanoseconds compared to `process.hrtime.bigint()`
+ * @property {boolean} [deadlineReached=false] - Whether the deadline has been reached. Should not be set by the
```

Comment:
If it should not be set by a user, let us not document it here? And if I understand it correct, it is just a way to transport the information to other code parts. Is there any other way to transport that information?